webpack/strategies/docs.js

```diff
@@ -23,10 +23,10 @@ export default (config, options) => {
         noParse: /babel-core\/browser/,
         loaders: config.module.loaders
           .map(value => {
-            if (/\.js\/$/.test(value.test.toString())) {
+            if (/js/.test(value.test.toString())) {
```

Comment:
Quick question, why removing the `\.` ? That means any filename with js in it (don't have any example in mind right now) would also get match?